.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,28 @@
+# .github/workflows/chromatic.yml
```

Comment:
Maybe we can remove most of these comments.
Most of the structure is self-explainatory